iOS/src/toga_iOS/widgets/webview.py

```diff
@@ -30,9 +36,39 @@ def _completion_handler(res: objc_id, error: objc_id) -> None:
     return _completion_handler
 
 
+def parse_color_rule(color_rule_str):
```

Comment:
see if you can pass in the bg_default_color to this

seems after generating web_view we can
```
self.web_view = toga.WebView()
self.web_view.style```

and try to get it using interface